src/gsy_e/models/strategy/external_strategies/__init__.py

```diff
@@ -417,7 +417,7 @@ def _bid_aggregator_impl(
             )
             response = {
                 "command": "bid", "status": "ready",
-                "bid": bid.to_json_string(replace_existing=replace_existing),
```

Comment:
Not really, this was one of the funny discoveries that I stumbled upon. This was doing nothing in the former implementation, because the `to_json_string` did not receive this argument. The implementer was probably confused here because this was a method argument of the `TwoSidedMarket.bid` method and not of the Bid class, neither of `Bid.to_json_string` method